extensions/ql-vscode/.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Since this is only used for Storybook, is it possible to have it in `/extensions/ql-vscode/.storybook`?

Reply:
Unfortunately, that doesn't work and it will not compile.

Reply:
The issue is that now there is a `.babelrc` file sitting at the top which could confuse people as there are so many elements to compilation of the extension. And with the PR being squashed merged we lose the easy access to the history. I don't have a good suggestion right now, but it's something to think about how we could make better.

Reply:
I agree that it's somewhat confusing, but I didn't squash merge this PR, so the original commit is now on `main` and shows why this was added: https://github.com/github/vscode-codeql/commit/bb05220b2c5acd906d9d61ba963aff17c3d9373b

Reply:
Oh nice sorry I misread the history!